### PR TITLE
Remove outdated pages

### PR DESCRIPTION
**Pull request by jmagman**
_Description of what this PR is changing or adding, and why:_ Remove outdated pages. These listings are no longer open.

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -8,9 +8,5 @@
     See how Flutter is pushing UI development at Flutter Forward;<br>
     streaming live from Kenya on January 25, 2023.
     <a href="https://flutter.dev/events/flutter-forward">Register now</a>
-    {% comment %}
-    <br>The Flutter and Dart teams are hiring.
-    <a href="https://docs.flutter.dev/jobs">Learn more</a><br>
-    {% endcomment %}
```
Comment (sfshaza2) [inline src/_includes/banner.html:14]: This is already commented out, so it doesn't even show up in the HTML as a comment. I feel no need to delete it.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
These files are not being displayed currently. Why do we need to remove them from the repo? Is it a case of people following these links and finding the pages? If so, I prefer a different solution. I would, for now, add a redirect that goes to a "we are not currently hiring." page.

**Comment by jmagman** [comment]
> These files are not being displayed currently. 

I navigated to them from https://docs.flutter.dev/jobs. I guess why would we not remove them from the repository since it's source controlled and we can bring it back whenever we want?

However you want to solve this is fine, but those postings should be removed.

**Comment by sfshaza2** [comment]
Closing. Replaced by PR https://github.com/flutter/website/pull/7973

**Comment by jmagman** [comment]
Thanks @sfshaza2!

**Pull request closed without merging**